server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -610,7 +610,7 @@ private class EmailFullViewFactory @Inject()(zoneIdProvider: ZoneIdProvider, pre
         .getOrElse(Failure(new IllegalArgumentException("No message supplied")))
       mime4JMessage <- Email.parseAsMime4JMessage(firstMessage)
       blobId <- BlobId.of(messageId)
-      bodyStructure <- EmailBodyPart.ofMessage(request.bodyProperties, zoneIdProvider.get(), blobId, mime4JMessage, firstMessage.getSize)
+      bodyStructure <- EmailBodyPart.ofMessage(request.bodyProperties, zoneIdProvider.get(), blobId, mime4JMessage, firstMessage.getBody.size)
```

Comment:
Was putting the full message size for the body one

Reduced failing tests from 5 to 1 on EmailGetMethodContract

There is still a problem with `bodyStructureForSimpleMultipart` test, size difference of 2 (actual 2285, expected 2287). Dont know if it's right and previous code wrong or if there is a little trick here.

Will check back after lunch

Reply:
Just a guess, maybe someone counts the CR+LF between headers and body towards the body size?

Reply:
Maybe there is something like that... Honestly could not put my finger on it.
Thanks for having a guess at it though :)

I think I will pretend it's acceptable. @chibenwa don't hesitate to disagree with this decision if you think it's better

Reply:
Quite likely.

However that's the size of the message as stored in the mailbox.

I suggest downloading the EML and checking the size and validating which on it the right one...

It is not impossible the the MessageWriter does some sanitizing like adding a trailing CRLF if missing...

Reply:
Might be that... seems to be 2285 to me when I download the eml and check the multipart/mixed part. Maybe some sanitizing at the work yes. So I'm fine with this honestly